### ISSUE TITLE
OhMyREPL does not play nice with the Rebugger

### ISSUE THREAD

**Issue by adamryczkowski** [KristofferC/OhMyREPL.jl#129]
There is a conflict between `OhMyREPL` and the `Rebugger`. At the moment one cannot have both at the same time. In short, whatever package gets loaded first, it wins.

If I load `OhMyREPL` _before_ `Rebugger`, then `Rebugger`'s keyboard shortcut will no longer work (and the package will be completely useless)

If I load `Rebugger`first, then `OhMyREPL` is crippled: it highlights syntax and rainbows brackets only after I press `backspace` or other non-character key in the REPL. Bracket completion and highlighting ceases to work completely. 

I have filled a separate issue on Rebugger as well: https://github.com/timholy/Rebugger.jl/issues/30

**Comment by KristofferC** [comment]
I don't think anyone really uses Rebugger so this is not a priority.
